oneview/provider.go

```diff
@@ -68,49 +68,52 @@ func Provider() terraform.ResourceProvider {
 			"oneview_interconnect_type":          dataSourceInterconnectType(),
 			"oneview_interconnect":               dataSourceInterconnects(),
+			"oneview_logical_enclosure":          dataSourceLogicalEnclosure(),
 			"oneview_logical_interconnect":       dataSourceLogicalInterconnect(),
 			"oneview_logical_interconnect_group": dataSourceLogicalInterconnectGroup(),
+			"oneview_network_set":                dataSourceNetworkSet(),
 			"oneview_scope":                      dataSourceScope(),
 			"oneview_server_hardware":            dataSourceServerHardware(),
 			"oneview_server_hardware_type":       dataSourceServerHardwareType(),
 			"oneview_storage_attachment":         dataSourceStorageAttachment(),
-			"oneview_logical_enclosure":          dataSourceLogicalEnclosure(),
-			"oneview_enclosure_group":            dataSourceEnclosureGroup(),
 			"oneview_server_profile":             dataSourceServerProfile(),
 			"oneview_server_profile_template":    dataSourceServerProfileTemplate(),
 			"oneview_storage_system":             dataSourceStorageSystem(),
-			"oneview_uplink_set":                 dataSourceUplinkSet(),
-			"oneview_network_set":                dataSourceNetworkSet(),
 			"oneview_storage_volume_template":    dataSourceStorageVolumeTemplate(),
+			"oneview_uplink_set":                 dataSourceUplinkSet(),
 			"oneview_volume":                     dataSourceVolume(),
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
 			"oneview_deployment_plan":            resourceDeploymentPlan(),
-			"oneview_server_profile":             resourceServerProfile(),
 			"oneview_enclosure":                  resourceEnclosure(),
 			"oneview_enclosure_group":            resourceEnclosureGroup(),
 			"oneview_ethernet_network":           resourceEthernetNetwork(),
-			"oneview_network_set":                resourceNetworkSet(),
 			"oneview_fcoe_network":               resourceFCoENetwork(),
 			"oneview_fc_network":                 resourceFCNetwork(),
-			"oneview_scope":                      resourceScope(),
-			"oneview_server_profile_template":    resourceServerProfileTemplate(),
-			"oneview_storage_system":             resourceStorageSystem(),
+			"oneview_hypervisor_manager":         resourceHypervisorManager(),
+			"oneview_i3s_plan":                   resourceI3SPlan(),
+			"oneview_logical_enclosure":          resourceLogicalEnclosure(),
 			"oneview_logical_interconnect_group": resourceLogicalInterconnectGroup(),
 			"oneview_logical_interconnect":       resourceLogicalInterconnect(),
 			"oneview_logical_switch_group":       resourceLogicalSwitchGroup(),
-			"oneview_uplink_set":                 resourceUplinkSet(),
-			"oneview_i3s_plan":                   resourceI3SPlan(),
-			"oneview_logical_enclosure":          resourceLogicalEnclosure(),
+			"oneview_network_set":                resourceNetworkSet(),
+			"oneview_scope":                      resourceScope(),
+			"oneview_server_certificate":         resourceServerCertificate(),
```

Comment:
ServerCertificate is not supported yet in this PR. Remove this entry.

Reply:
Done